antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4

```diff
@@ -83,7 +83,7 @@ storageGroupAttributesClause
     ;
 
 storageGroupAttributeClause
-    : (TTL | SCHEMA_REPLICATION_FACTOR | DATA_REPLICATION_FACTOR | TIME_PARTITION_INTERVAL) '=' INTEGER_LITERAL
+    : (TTL | SCHEMA_REPLICATION_FACTOR | DATA_REPLICATION_FACTOR | TIME_PARTITION_INTERVAL | SCHEMA_REGION_GROUP_NUM | DATA_REGION_GROUP_NUM) '=' INTEGER_LITERAL
```

Comment:
Please update the Doc at the same time:
https://iotdb.apache.org/zh/UserGuide/Master/Operate-Metadata/Database.html#%E6%9F%A5%E7%9C%8B%E6%95%B0%E6%8D%AE%E5%BA%93
https://iotdb.apache.org/UserGuide/Master/Operate-Metadata/Database.html#%E6%9F%A5%E7%9C%8B%E6%95%B0%E6%8D%AE%E5%BA%93
The doc is important for user